crates/pallet-domains/src/domain_registry.rs

```diff
@@ -82,6 +65,12 @@ pub struct DomainObject<Number, AccountId> {
     pub genesis_receipt_hash: ReceiptHash,
     /// The domain config.
     pub domain_config: DomainConfig,
+    /// The genesis config of the domain, encoded in json format.
+    //
+    /// NOTE: the WASM code in the `system-pallet` genesis config should be empty to avoid
+    /// redundancy with the `RuntimeRegistry`. Currently, this value only set to `Some` for
+    /// the genesis domain instance.
+    pub raw_genesis_config: Option<Vec<u8>>,
```

Comment:
IMHO This will not be used for any production genesis domains. We should make the comment very clear of that

Reply:
Currently, this is set to `Some` for the genesis domain of both `dev`, `gemini`, `devnet` and `local` network (plz take a look at [this function](https://github.com/subspace/subspace/pull/1704/files#diff-6301df3409749688d1988db7fcff349b3e77245d5635fe3a5e24889b6f19cf46R167)), it is mainly for keeping the domain's genesis config as the same as in v1, because in v1 the `gemini` and `devnet` network have also set the genesis relayer and sudo account, I can change this if it is unexpected.

Reply:
I understand, so this is used by the client to recontruct the chainspec ?

Reply:
Right